src/software/ai/evaluation/time_to_travel_test.cpp

```diff
@@ -0,0 +1,208 @@
+#include "software/ai/evaluation/time_to_travel.h"
+
+#include <gtest/gtest.h>
+
+#include "software/test_util/equal_within_tolerance.h"
+
+class TimeToTravel : public ::testing::Test
+{
+   protected:
+    RobotConstants_t robot_constants = create2015RobotConstants();
+};
+
+TEST_F(TimeToTravel, getTimeToTravelDistance_already_at_dest)
+{
+    EXPECT_EQ(Duration::fromSeconds(0), getTimeToTravelDistance(0, 2.0, 3.0));
+}
+
+TEST_F(TimeToTravel, getTimeToTravelDistance_reaches_max_velocity)
+{
+    // Check that the robot reaches the dest in the at the expected time when
```

Comment:
nit: "reaches the dest at the expected time"

Reply:
Updated